src/index.ts

```diff
@@ -66,9 +69,7 @@ export const fetcher = <Config extends Payload, Error>(
   baseURL: string,
   url: string,
   method: string
-): Fetcher<Config, Error> & {
-  abort: AbortController['abort'];
-} => {
+): Fetcher<Config, Error> & EffectAbort => {
   let abortController: AbortController | undefined;
 
   return Object.assign(
```

Comment:
This `assign` can be removed?

Reply:
I think can't be removed because fetcher should return callback and abort method for the line 132 
<img width="867" alt="image" src="https://github.com/neruchev/types-to-fetchers/assets/61740164/e23185cc-5e54-443d-856b-c70bd1496b18">